contrib/install_db4.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
```

Comment:
Should be part of the tarball / `make dist` (`DIST_CONTRIB` in Makefile.am)

Reply:
Done, thanks!